dockers/docker-sonic-telemetry/telemetry_vars.j2

```diff
@@ -1,5 +1,5 @@
 {
-    "x509" : "{% if DEVICE_METADATA['x509'] is defined %}{{ DEVICE_METADATA['x509'] }}{% endif %}",
-    "gnmi" : "{% if DEVICE_METADATA['gnmi'] is defined %}{{ DEVICE_METADATA['gnmi'] }}{% endif %}",
-    "certs": "{% if DEVICE_METADATA['certs'] is defined %}{{ DEVICE_METADATA['certs'] }}{% endif %}"
-}
+    "certs": "{% if "certs" in TELEMETRY.keys() %}{{ TELEMETRY["certs"] }}{% endif %}",
+    "gnmi" : "{% if "gnmi" in TELEMETRY.keys() %}{{ TELEMETRY["gnmi"] }}{% endif %}",
+    "x509" : "{% if "x509" in DEVICE_METADATA.keys() %}{{ DEVICE_METADATA["x509"] }}{% endif %}"
+}
```

Comment:
Add newline back at EOF.

Reply:
@jleveque can you please resolve this request? This is blocking the merge at this point of time.